src/pkg/bundle/create.go

```diff
@@ -107,3 +114,60 @@ func (b *Bundle) confirmBundleCreation() (confirm bool) {
 	}
 	return true
 }
+
+// processValuesFiles reads values from valuesFiles and updates the bundle with the override values
+func (b *Bundle) processValuesFiles() error {
+	// Populate values from valuesFiles if provided
+	for i, pkg := range b.bundle.Packages {
+		for j, overrides := range pkg.Overrides {
+			for k, bundleChartOverrides := range overrides {
+				for _, valuesFile := range bundleChartOverrides.ValuesFiles {
+					// Check relative vs absolute path
+					fileName := filepath.Join(b.cfg.CreateOpts.SourceDirectory, valuesFile)
+					if filepath.IsAbs(valuesFile) {
+						fileName = valuesFile
+					}
+					// read values from valuesFile
+					values, err := chartutil.ReadValuesFile(fileName)
+					if err != nil {
+						return err
+					}
+					if len(values) > 0 {
+						// populate BundleChartValue slice to use for merging existing values
+						valuesFileValues := make([]types.BundleChartValue, 0, len(values))
+						for key, value := range values {
+							valuesFileValues = append(valuesFileValues, types.BundleChartValue{Path: key, Value: value})
+						}
+						// update bundle with override values
+						overrides := b.bundle.Packages[i].Overrides[j][k]
+						// Merge values from valuesFile and existing values
+						overrides.Values = mergeBundleChartValues(overrides.Values, valuesFileValues)
+						b.bundle.Packages[i].Overrides[j][k] = overrides
+					}
+				}
+			}
+		}
+	}
+	return nil
+}
+
+// mergeBundleChartValues merges two lists of BundleChartValue using the values from list1 if there are duplicates
+func mergeBundleChartValues(list1, list2 []types.BundleChartValue) []types.BundleChartValue {
```

Comment:
Nice! My only recommendation is to have the second arg override the first (I think this is more standard), and rename `list1` and `list2` to reflect what var is going to override the other (`originals` vs `overrides` or something like that)